Assets/Scenes/ManaScene.unity

```diff
@@ -523,6 +523,8 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   ManaBarFill: {fileID: 1293296639}
+  adjustMana: 0
+  lerpDuration: 1.5
```

Comment:
a bit long, maybe make it 0.5 seconds snappier visuals feel better especially for important information